### PR TITLE
Remove C dependencies

### PR DESCRIPTION
**Pull request by zweihander**
Hello! I was looking for a project for Hacktoberfest, and found your repository :)
PR removes [git2go](https://github.com/filhodanuvem/git2go) dependency and uses [go-git](https://github.com/go-git/go-git) instead (which was written in pure go).
This brings back easy cross compilation and all the stuff which gophers love.
I haven't tested it heavily, but it looks like the queries from Readme are working correctly.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,18 +1,89 @@
+github.com/alcortesm/tgz v0.0.0-20161220082320-9c5fe88206d7 h1:uSoVVbwJiQipAclBbw+8quDsfcvFjOpI5iCf4p/cqCs=
```
Comment (filhodanuvem) [inline go.sum:1]: I'm impressed about the amount of dependencies of go-git, it seems ok for you ?

**Review by filhodanuvem — CHANGES_REQUESTED** [review]

**Comment by filhodanuvem** [comment]
Thanks for the help @zweihander , it looks promising 😍 .

File: go.sum (hunk 1 above)
Comment (zweihander) [reply go.sum:1]: Apparently the authors of the go-git did not implement some functionality and used the libraries created by the community. I don't see anything wrong with that.

File: runtime/commits.go [hunk 2]
```diff
@@ -2,19 +2,24 @@ package runtime
...
-	builder.walk.PushHead()
-	builder.walk.Sorting(git.SortTime)
+	iter, err := repo.CommitObjects()
+	if err != nil {
+		return nil, err
+	}
+
+	// builder.walk, _ = repo.Walk()
+	// builder.walk.PushHead()
+	// builder.walk.Sorting(git.SortTime) <- ?
```
[the remark this answers is not in the document]
Comment (zweihander) [reply runtime/commits.go]: fixed

**Comment by filhodanuvem** [comment]
Thanks for the effort @zweihander 🎉

**Comment by zweihander** [comment]
Yes, I'll do it :wink:

**Comment by filhodanuvem** [comment]
Nevermind @zweihander , Just did it oin #96 because the CI has broken. 
I just don't know why it has failed just after merge 🤔

**Comment by zweihander** [comment]
Seems new gh actions is ok

**Pull request merged**